api/go.mod

```diff
@@ -1,13 +1,8 @@
 module jester.noble.xyz/api
 
-go 1.24
+go 1.22.11
```

Comment:
In theory we could leave this as just 1.22 no?

Reply:
Since there is no issue,  will follow up with changelog in separate commit.